node/src/actors/peers_manager/actor.rs

```diff
@@ -67,7 +68,11 @@ impl Actor for PeersManager {
                 act.persist_peers(ctx, storage_peers_period);
 
                 // Start the feeleer peers process on SessionsManager start
-                act.feeler(ctx, feeler_peers_period);
+                act.feeler(
+                    ctx,
+                    feeler_peers_period,
+                    Duration::from_secs(config.consensus_constants.checkpoints_period as u64),
```

Comment:
Why is this offset needed?

Reply:
Same question here :thinking:

Reply:
> negligible

Another way is to check if there are a difference of more than one epochs to avoid this

Reply:
Some problems appears when we try to send messages from a session which have not current_epoch updated (and it has a None). So this offset is to create a wait at the beginning, when the node is starting, to do not use the feeler and the bootstrap methods until have an updated Epoch in current_epoch state

Reply:
So we wouldn't need that if we had a synchronous way to get the current epoch at the beginning? Maybe we could use `EpochConstants::epoch_at(get_timestamp())`, and get `EpochConstants` directly from config.